dimager/views.py

```diff
@@ -1,3 +1,109 @@
 from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from dimager.models import ImagerProfile
+from imager_images.models import Album, Photo, User
+from django.views.generic import CreateView, UpdateView, DeleteView
 
-# Create your views here.
+
+@login_required()
+def user_profile(request, id):
+    user = User.objects.get(id=id)
+    try:
+        albums = Album.objects.filter(user=user).all()
+    except:
+        albums = None
+    profile = ImagerProfile.objects.get(user=user)
```

Comment:
Again, your `user` object has an attribute that returns the profile for that user?  No?  Just use that.  Don't work harder than you have to.